src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -727,9 +729,15 @@ To read an explanation of what these settings are and how they relate to each ot
 
 You can set the [aggregation window duration](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/streaming-alerts-key-terms-concepts#aggregation-window) to choose how long data is accumulated in a streaming time window before it's aggregated. You can set it to anything between 30 seconds and 15 minutes. The default is one minute.
 
-<Callout variant="tip">
-  Baseline alert condition thresholds don't support editing the aggregation window. They use the 1 minute default.
-</Callout>
+### Sliding window aggregation [#sliding-window-aggregation]
+
+You can use [sliding windows](/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/create-smoother-charts-sliding-windows) to create smoother charts. This is done by creating overlapping windows of data.
+
+Once enabled, set the slide by duration to control how much overlap time your aggregated windows have. The slide by duration must divide equally into the aggregation window duration and also be less than that duration.
```

Comment:
How about "The slide by duration must be shorter than the aggregation windows duration, and must divide equally between them."

Reply:
Yeah, I like this better. Thanks!